shims/spark311/src/main/scala/org/apache/spark/sql/rapids/shims/spark311/GpuInMemoryTableScanExec.scala

```diff
@@ -28,87 +28,86 @@ import org.apache.spark.sql.execution.{LeafExecNode, SparkPlan}
 import org.apache.spark.sql.execution.columnar.InMemoryRelation
 import org.apache.spark.sql.vectorized.ColumnarBatch
 
-case class GpuInMemoryTableScanExec(
-  attributes: Seq[Attribute],
-  predicates: Seq[Expression],
-  @transient relation: InMemoryRelation) extends LeafExecNode with GpuExec {
-
-    override val nodeName: String = {
-      relation.cacheBuilder.tableName match {
-        case Some(_) =>
-          "Scan " + relation.cacheBuilder.cachedName
-        case _ =>
-          super.nodeName
-      }
+case class GpuInMemoryTableScanExec(attributes: Seq[Attribute],
+   predicates: Seq[Expression],
+   @transient relation: InMemoryRelation) extends LeafExecNode with GpuExec {
```

Comment:
4-space indent for continuations
```suggestion
    predicates: Seq[Expression],
    @transient relation: InMemoryRelation) extends LeafExecNode with GpuExec {
```